install/installer/docs/overview.md

```diff
@@ -0,0 +1,589 @@
+and did not have effective validation on it. By contrast, the Installer's
+config is < 50 lines long and can be fully validated before running.
+
+Also, by baking-in the container image versions to each release of the
+Installer, we reduce the potential for variance making it easier to support
+the community.
+
+## How do I use Cert Manager to create a TLS certificate?
+
+> Please see [cert-manager.io](https://cert-manager.io) for full documentation.
+> This should be considered a quickstart guide.
+
+There are two steps to creating a public TLS certificate using cert-manager.
+
+### 1. Create the Issuer/ClusterIssuer
+
+As the certificate is a wildcard certificate, you must use the DNS01 Challenge
+Provider. Please consult their [documentation](https://cert-manager.io/docs/configuration/acme/dns01)
+for instructions. This can be either an
+[`Issuer` or `ClusterIssuer`](https://cert-manager.io/docs/concepts/issuer).
+
+### 2. Create the certificate
+
+Replace `$DOMAIN` with your domain. This example assumes you have created a
+`ClusterIssuer` called `gitpod-issuer` - please change this if necessary.
+
+This certificate is called `https-certificates` - please use that in your
+Gitpod installer [config](#config).
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: https-certificates
+spec:
+  secretName: https-certificates
+  issuerRef:
+    name: gitpod-issuer
+    kind: ClusterIssuer
+  dnsNames:
+    - $DOMAIN
+    - "*.$DOMAIN"
+    - "*.ws.$DOMAIN"
+```
+
+## How do I use my own TLS certificate?
+
+If you don't wish to use cert-manager to create a TLS certificate with a public
+certificate authority, you can bring your own.
+
+To do this, generate your certificate as you would normally and then create a
+secret with the CRT set to `tls.crt` and the Key set to `tls.key`.
+
+The DNS names must be `$DOMAIN`, `*.$DOMAIN` and `*.ws.$DOMAIN`, where `$DOMAIN`
+is your domain.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: https-certificates
+data:
+  tls.crt: xxx
+  tls.key: xxx
+```
+
+## How can I install to a Kubernetes namespace?
+
+By default, Gitpod will be installed to the `default`
+[Kubernetes namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces).
+To install to a different namespace, pass a `namespace` flag to the `render`
+command.
+
+```shell
+gitpod-installer render --config gitpod.config.yaml --namespace gitpod > gitpod.yaml
+```
+
+The `validate cluster` command also accepts a namespace, allowing you to run
+the checks on that namespace.
+
+```shell
+gitpod-installer validate cluster --kubeconfig ~/.kube/config --config gitpod.config.yaml --namespace gitpod
+```
+
+**IMPORTANT**: this does not create the namespace, so you will need to create
+that separately. This is so that uninstallation of Gitpod does not remove any
+Kubernetes objects, such as your TLS certificate or connection secrets.
+
+```shell
+kubectl create namespace gitpod
+```
+
+# Todo
```

Comment:
This subheading and the stuff below may as well be removed now - not the end of the world if it's not though

Reply:
Removing it, thanks!